src/Nri/Ui/Button/V8.elm

```diff
@@ -154,6 +154,24 @@ button :
         }
     -> Html msg
 button config content =
+    customizableButton config content []
+
+
+{-| -}
+customizableButton :
```

Comment:
💭 rather than adding this as a new extra function, I wonder if now is the time to change the Button API to the pattern of taking attributes?  Roughly like...

```elm
module ...Button.V9

type Attribute
    = Size ButtonSize
    | Style ButtonStyle
    | ExtraAttribute Sytyled.Attribute msg
    | Width ButtonWidth
    ...


button : msg -> List Attribute -> Html msg
link : Url -> List Attribute -> Html msg
```

Then all the attributes would be optional, and new flexibility like this customizable button would fit into the existing API rather than requiring a new function to be added.

Reply:
Oooooo yes!!! Love this idea